### DISCUSSION TITLE
Updating to the Latest Tag When Homebrew's in Developer Mode

### DISCUSSION THREAD

**Discussion by RandomDSdevel** [Homebrew/discussions#1315] [Everyday usage]
Continuing off of Homebrew/brew#11189 (@i0ntempest:)  

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;You can also get this behavior when Git configuration variable '`homebrew.devcmdrun`' or shell environment variable '`HOMEBREW_DEVELOPER`' is set by setting the '`HOMEBREW_UPDATE_TO_TAG`' environment variable.  I use this and have a '`stable`' branch locally, and this works seamlessly:  '`brew update`' updates Homebrew/brew's '`master`' branch to its latest commit, then switches back to '`stable`' and checks out the latest stable, tagged release there.  (If I remember correctly, there may actually have _been_ a '`stable`' branch up here upstream on GitHub at some point before both the core/formula split and Homebrew v1.0.0, so that may be why you have a '`stable`' branch set up locally; it got cloned back then.  I can't remember exactly when this was, though.)

**Comment by carlocab** [comment]
Forgot about `HOMEBREW_UPDATE_TO_TAG`. That's actually the solution you want, @i0ntempest. Thanks, @RandomDSdevel.

**Reply by RandomDSdevel** [reply]
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No problem, @carlocab!
